council/llm/anthropic_llm_configuration.py

```diff
@@ -1,15 +1,20 @@
 from __future__ import annotations
 from typing import Optional
 
-from council.utils import read_env_str
+from council.utils import read_env_str, read_env_float
+from council.llm.llm_configuration_base import _DEFAULT_TIMEOUT
 
 
 class AnthropicLLMConfiguration:
     """
     Configuration for Anthropic LLMs
     """
 
-    def __init__(self, model: Optional[str] = None, api_key: Optional[str] = None):
+    timeout: float
+    model: str
+    api_key: str
```

Comment:
those are dangerous static variables which should be avoided, in favor of instance variables on `self.`